src/pages/Index.vue

```diff
@@ -142,7 +142,7 @@ body
 <script>
 export default {
   mounted() {
-    if (this.$q.localStorage.getItem('server') === null) {
+    if (this.$q.localStorage.getItem('server') === undefined) {
```

Comment:
When I test the frontend on the server or my local machine (with Chrome), the value in the `localStorage` is initially `null`. Have you observed `undefined`?

Reply:
Actually, I don't know where this is coming from.
I was just applying some i18n

Reply:
I can change that to null